reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -134,23 +129,13 @@ data class EvaluatedModel(
             ScopeExclude::class.java
         )
 
-        private val MAPPER_CONFIG: ObjectMapper.() -> Unit = {
-            registerKotlinModule()
-
-            registerModule(JavaTimeModule())
-            registerModule(IntIdModule(INT_ID_TYPES))
-
-            disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-
-            propertyNamingStrategy = PROPERTY_NAMING_STRATEGY
-        }
-
         private val JSON_MAPPER by lazy {
-            val factory = JsonFactory().disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)
-            JsonMapper(factory).apply(MAPPER_CONFIG)
+            jsonMapper.registerModule(IntIdModule(INT_ID_TYPES)).disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)
```

Comment:
Before the mappers where independent, now they are shared.
Why is it ok to use a shared instance in particular since it is modified via the `disable()`` call?

Reply:
It's *not* ok 😁 Thanks for pointing this out, fixed.